tfexec/errors.go

```diff
@@ -5,6 +5,20 @@ import (
 	"fmt"
 	"os/exec"
 	"regexp"
+	"strings"
+)
+
+var (
+	// The "Required variable not set:" case is for 0.11
```

Comment:
Moved these out here so that the bad regexp panic happens at `init` time, and not at error parse time.